ako-operator/helm/ako-operator/values.yaml

```diff
@@ -64,6 +73,7 @@ L7Settings:
   shardVSSize: "LARGE" # Use this to control the layer 7 VS numbers. This applies to both secure/insecure VSes but does not apply for passthrough. ENUMs: LARGE, MEDIUM, SMALL, DEDICATED
   passthroughShardSize: "SMALL" # Control the passthrough virtualservice numbers using this ENUM. ENUMs: LARGE, MEDIUM, SMALL
   noPGForSNI: false # Switching this knob to true, will get rid of poolgroups from SNI VSes. Do not use this flag, if you don't want http caching. This will be deprecated once the controller support caching on PGs.
+  enableMCI: "false" # Enabling this flag would tell AKO to start processing multi-cluster ingress objects.
```

Comment:
@DixitAakash: Remove `enableMCI` from everywhere.

Reply:
Removed from AKOConfig CRD Spec but added to ako configmap.